scopes/component/lister/list.cmd.ts

```diff
@@ -16,16 +16,16 @@ type ListFlags = {
 
 export class ListCmd implements Command {
   name = 'list [remote-scope]';
-  description = 'list components on a workspace, local scope or a remote scope.';
+  description = 'list components on a workspace, local scope or a remote scope.'; // @david what's the difference between workspac and local scope? Can we just remove local scope in this description?
```

Comment:
@benjgil 
workspace = bitmap
local scope - what you have in the .bit objects (it might contain many components which are not in bitmap (like deps, envs, aspects)
also it might not contain some comps from the bitmap (like new components which not snapped yet)

Reply:
OK, but the list command will list either the local scope (dont think it does) or the workspace. There's no way i can see to choose between them.
So which is going to be shown here when i run `bit list` without the remote flag? (my money's on workspace)

Reply:
Yes, it'll show the workspace.
If you added `--scope`, it'll show the local scope. (so not including new)

Reply:
then i'm going to change the description to just "... on a workspace or a remote scope" and the --scope flag is something that whoever needs it will use it - having it in the description just confuses most users